spaceship/lib/spaceship/client.rb

```diff
@@ -89,6 +89,10 @@ def teams
       end
     end
 
+    def self.base_url
+      ENV["ITC_ITUNES_ENDPOINT"] ? "itunesconnect.apple.com" : "appstoreconnect.apple.com"
```

Comment:
Why the ENV variable? 
What problem does this solve that the endpoint can be switched?

Reply:
#13282 was also caused by "Service key is empty", which was triggered by the wrong URL being used on the olympus.itunes.apple.com request. 1.103.1 should fix that.